src/irmin-pack/unix/mapping_file.ml

```diff
@@ -327,6 +338,35 @@ let calculate_extents_oc ~(src_is_sorted : unit) ~(src : int_bigarray)
   in
   dst_off
 
+let rev_inplace (src : int_bigarray) : unit =
```

Comment:
I think it would be nice for future readers to have a short description of both `rev_inplace` and `set_prefix_offsets` since they work together.  For instance, it is not immediately clear why the reverse does not move the middle item, but is clear if you know it is 0 when this is called and will be filled in by `set_prefix_offsets`.